src/java/org/apache/cassandra/repair/autorepair/AutoRepairUtils.java

```diff
@@ -373,12 +385,6 @@ public static void setForceRepair(RepairType repairType, UUID hostId)
         logger.info("Set force repair repair type: {}, node: {}", repairType, hostId);
     }
 
-    public static CurrentRepairStatus getCurrentRepairStatus(RepairType repairType)
```

Comment:
Was only used by tests so removed this and updated test to use the other `getCurrentRepairStatus`